### PR TITLE
Space-Traveller Hub

### PR DESCRIPTION
**Pull request by felixDev22**
Hi ✋ 

We were two in the project:
@ameerhamzakhalid 
@felixDev22 

We have implemented the requirements as follows:

- [ ]  Add Rocket Component (default)
- [ ]  Fetched the Rocket API
- [ ]  Added Rocket Reducer connected with API
- [ ]  Add Rocket Reserve Button Functionality
- [ ]  Fetched the Mission API
- [ ]  Added Mission Component
- [ ]  Added Mission Reducer connected with API
- [ ]  Added Mission Reserve Button Functionality
- [ ]  Added Navigation Links
- [ ]  Added My Profile Component and connect with Rocket and Mission
- [ ]  Added Required Styles to the pages
- [ ]  Updated README documentation
- [ ]  Checked that the file is working properly

### CODE REVIEW

File: src/components/pages/Rockect.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+/* eslint-disable react/prop-types */
```
Comment (OLIPLICHE) [inline src/components/pages/Rockect.js]: - Proptypes is a mechanism that ensures that the passed value is of the correct datatype. I think you should not disable the Linter, you can solve it manually.

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi Team 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Gitflow is correctly used (

## Required Changes ♻️
- Write unit tests with React Testing Library and Jest(for unit tests). This is because it is part of the project requirements
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/pages/Rockect.js (hunk 1 above)
Comment (ameerhamzakhalid) [reply src/components/pages/Rockect.js]: done

**Review by roshan-bajgain — APPROVED** [review]
Hi @felixDev22 and @ameerhamzakhalid, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**